src/ledger/LedgerManagerImpl.cpp

```diff
@@ -385,6 +400,23 @@ LedgerManagerImpl::loadLastKnownLedger(bool restoreBucketlist)
             // Work should only fail during graceful shutdown
             releaseAssertOrThrow(mApp.isStopping());
         }
+        if (mLedgerStateCache)
+        {
+            auto populateLedgerCacheWork =
+                mApp.getWorkScheduler().executeWork<PopulateLedgerCacheWork>();
```

Comment:
I think this would be better as a step in `AssumeStateWork`. That's where we do all our "in-memory matches current state" work, such as setting the BucketList, indexing buckets, etc.